searx/search/processors/abstract.py

```diff
@@ -154,6 +154,11 @@ def get_params(self, search_query, engine_category):
         if search_query.time_range and not self.engine.time_range_support:
             return None
 
+        # if max page is reached, skip
+        max_page = self.engine.max_page or settings['search']['max_page']
+        if max_page and max_page < search_query.pageno:
+            return None
```

Comment:
Nitpicking: is it possible to group the two tests about `search_query.pageno` together so it is easy to see there are two tests about search_query.pageno ? (basically just move lines 157 - 160 above line 153).
Nitpicking...